README.md

```diff
@@ -7,10 +7,10 @@ Used by [FoE Bot](https://github.com/m3talstorm/foe-bot) for request signature g
 *Purely educational and to fuel curiosity*
 
 ## Summary
-- Binary: https://foeen.innogamescdn.com/swf/Main.swf?1511779469
-- Version: 1.116
-- Timestamp: 1513162258
-- Secret: Ob90uMKvbFGtB7/yWM19s51P/hOnbg289yT4V7qHTs8qTr/vefAMsYesk21tRIxi7Gx/9byISR/a3nqz53QuyQ==
+- Binary: https://foeen.innogamescdn.com/swf/Main.swf
+- Version: 1.119
+- Timestamp: 05.02.2018 11:12
```

Comment:
Can we keep the timestamp in the same format, it makes it easier to marry up requests.

Reply:
I copied the timestamp from  the swf. dont think it has it in unix format, but will check